src/plugin/index.js

```diff
@@ -5,8 +5,20 @@ const importCwd = require("import-cwd"); // used to get the users project detail
 
 const pkg = importCwd("./package.json");
 
-function Jarvis(options) {
-  // TOOD: add options for port, etc..
+function Jarvis(options = {}) {
+  
+  // check if port is valid
+  let portIsValid = true;
```

Comment:
Since natively, nodeJS server will report an error if the port is not a number or if it's out of range we can remove this bit and delegate that to the server code.

and the function will only become:

```

function Jarvis(options = {}) {
  this.options = {
    port: options.port || 1337 // fall back to 1337 if port is not a number
  };

/* rest of code */
}
```